extensions/wikia/AdEngine/js/DOMElementTweaker.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define('ext.wikia.adEngine.domElementTweaker', [
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
+], function (log, doc, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.domElementTweaker',
+		rclass = /[\t\r\n]/g;
+
+	function removeClass(element, cls) {
+		var oldClasses,
+			newClasses = ' ' + element.className.replace(rclass, ' ') + ' ';
```

Comment:
consider using https://developer.mozilla.org/en/docs/Web/API/Element/classList although I don't totally understand what is going on there.